neo/Prompt/Commands/Wallet.py

```diff
@@ -196,16 +196,19 @@ def __init__(self):
 
     def execute(self, arguments):
         from_addr_str = None
+        to_addr_str = None
 
         args = arguments
         if args:
             args, from_addr_str = PromptUtils.get_from_addr(args)
+            args, to_addr_str = PromptUtils.get_to_addr(args)
 
-        return ClaimGas(PromptData.Wallet, True, from_addr_str)
+        return ClaimGas(PromptData.Wallet, True, from_addr_str, to_addr_str)
 
     def command_desc(self):
         p1 = ParameterDesc('--from-addr', 'source address to claim gas from (if not specified, take first address in wallet)', optional=True)
-        return CommandDesc('claim', 'claim gas', params=[p1])
+        p2 = ParameterDesc('--to-addr', 'destination address for claimed gas (if not specified, use the default change address; or, use the from address, if specified)', optional=True)
```

Comment:
```suggestion
        p2 = ParameterDesc('--to-addr', 'destination address for claimed gas (if not specified, take first address in wallet; or, use the from address, if specified)', optional=True)
```
let's use the same phrasing as `p1`